### ISSUE TITLE
Help pls

### ISSUE THREAD

**Issue by Teals53** [Dliwk/bs2ba#1]
how can i use this converter ? i wanna convert all mrmaxmeier scripts to new api

**Comment by Dliwk** [comment]
For the first, clone this repository.
You can get some help following way: `python -m bs2ba --help`
`python -m bs2ba mySuperAwesomeMod.py > my_super_awesome_mod.py` will create new patched file 'my_super_awesome_mod.py'. In the most cases this file isn't ready for use BombSquad 1.5 mod, you need to fix some errors yourself.
This tool can't correct patch mod what redefines some methods. Create a 'plugin' and redefine needed functions (they can change location) yourself.

**Comment by Teals53** [comment]
can u update old mrmaxmeier scripts for the new api ? so everyone can use that ? bcs im not good at on coding if there have some errors who need manual fix after patch

**Comment by Dliwk** [comment]
No, this tool isn't magic, it can't patch any mod itself. Ask programmers you know.

I can try patch some Mrmaxmeier mods when I have free time if you have no one to ask.

**Comment by Teals53** [comment]
except these ones
![image](https://user-images.githubusercontent.com/13446881/95662587-a771d600-0b40-11eb-9e91-0db0a9295c2e.png)

bcs these ones are already broken and not working aswell
